.gitlab-ci.yml

```diff
@@ -163,15 +163,20 @@ variables:
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
   DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v32377621-ce42cb8e
+  DATADOG_AGENT_BUILDIMAGES: v32778237-8467c81e
   DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_WINBUILDIMAGES: v32377621-ce42cb8e
+  DATADOG_AGENT_WINBUILDIMAGES: v32778237-8467c81e
   DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_ARMBUILDIMAGES: v32377621-ce42cb8e
+  DATADOG_AGENT_ARMBUILDIMAGES: v32778237-8467c81e
   DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v32377621-ce42cb8e
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v32778237-8467c81e
+  DATADOG_AGENT_KERNEL_MATRIX_TESTING_BUILDIMAGES_SUFFIX: ""
+  DATADOG_AGENT_KERNEL_MATRIX_TESTING_BUILDIMAGES: v32778237-8467c81e
+  DATADOG_AGENT_NIKOS_BUILDIMAGES_SUFFIX: ""
+  DATADOG_AGENT_NIKOS_BUILDIMAGES: v32778237-8467c81e
```

Comment:
Is it used somewhere else in the agent ?

Reply:
No, and I removed it in #24638